extra-web/httpd/autobuild/postinst

```diff
@@ -1,2 +1,29 @@
-chown -R 25:25 "$PKGDIR"/srv/www
-chown -R 25:25 "$PKGDIR"/srv/http
+#!/bin/env bash
+
+# Warn users that we no longer use `/srv/http' for default webroot.
+
+if [ -e "/srv/http" ] ; then
+	if [ $(ls -A /srv/http) ]; then
+		echo "Warning - we are no longer using \`/srv/http' for default web root."
+		echo "Consider moving them to \`/srv/www' instead."
+	fi
+fi
+
+# Enable default site
+# For Apache httpd the configuration files must be ended with .conf.
+if [ ! -e /etc/httpd/sites-enabled ]; then
+	mkdir -p /etc/httpd/sites-enabled
```

Comment:
Why not build it into the package? Again, operations like this won't be correctly applied when generating a system release using `aoscboostrap`, as it strips out out-of-package files (barring exceptions).